todo_app/templates/single_item.html

```diff
@@ -0,0 +1,18 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block content %}
+  <div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Items</h2>
+      <ul class="list-group mb-4">
+			<li>{{ item.id }} - {{ item.title }} - {{ item.status }}</li>
+      </ul>
```

Comment:
Bit misleading to use `<ul>` and `<li>` elements - they stand for "unordered list" and "list item"